style.css

```diff
@@ -721,3 +721,143 @@ button:active {
     display: none;
   }
 }
+
+/* -------CSS FOR POPUP MOBILE    */
+
+.wrap-pop {
+  background-color: rgb(0, 0, 0, 0.5);
+  display: none;
+  transition: 500ms ease-in-out;
+  padding: 2%;
+  height: 95vh;
+  border: 1px solid #ddd;
+  overflow-y: scroll;
+}
+
+.wrap-pop.active {
+  display: flex;
+  justify-content: center;
+  position: fixed;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  right: 0;
+  z-index: 10;
+}
+
+.popup-mb {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  background-color: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  padding: 4%;
+  margin: 2%;
+  transition: 200ms ease-in-out;
+  overflow-y: scroll;
+  overflow: hidden;
+}
```

Comment:
- [ ] Kindly I can't see the close button, kindly add it to match the Figma design.
- [ ] Kindly I can't see the tags and the paragraph, kindly add them to match the Figma design.
- [ ] Kindly make the pic take the whole width, to match the Figma design.

Your design
![image](https://user-images.githubusercontent.com/35535130/175435275-016dba6d-fcda-4ee3-86ef-4fa97696cc82.png)

Figma design
![image](https://user-images.githubusercontent.com/35535130/175435362-1cba460f-b63b-4e5a-86e4-2f14840938d1.png)